MAUI/TextInputLayout/Container-Type.md

```diff
@@ -0,0 +1,163 @@
+---
+layout: post
+title: Container Type in .NET MAUI Text Input Layout control | Syncfusion
+description: Learn here all about Container Type support in Syncfusion .NET MAUI Text Input Layout (SfTextInputLayout) control and more.
+platform: maui
+control: SfTextInputLayout
+documentation: ug
+---
+
+# Container Type in .NET MAUI Text Input Layout (SfTextInputLayout)
+
+Containers improve the discoverability of input view by creating a contrast between the input view and assistive elements.
+
+## Filled
+
+The background of the input view will be filled with container color, and its stroke (at the bottom edge) color and thickness will be changed based on its states. It can be enabled by setting the `ContainerType` property to `Filled`.
+
+{% tabs %} 
+
+{% highlight xaml %} 
+
+<inputLayout:SfTextInputLayout Hint="Name"
+                               ContainerType="Filled">
+    <Entry Text="John" />
+</inputLayout:SfTextInputLayout>  
+
+{% endhighlight %}
+
+{% highlight C# %} 
+
+var inputLayout = new SfTextInputLayout();
+inputLayout.Hint = "Name";
+inputLayout.ContainerType = ContainerType.Filled;
+inputLayout.Content = new Entry() { Text = "John" }; 
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![Filled type](images/ContainerType/Filled_Focused.jpg)
+
+## Outlined
+
+When setting the `ContainerType` property to `Outlined`, the container will be covered with a rounded-corner border.
```

Comment:
When setting the `ContainerType` property to `Outlined,` the container will be covered with a rounded-corner border.

Reply:
Addressed the changes